src/Cli/Command/RunCommand.php

```diff
@@ -72,7 +75,18 @@ protected function execute(InputInterface $input, OutputInterface $output)
     {
         $config = $this->getConfig(Path::makeAbsolute($input->getArgument('config'), getcwd()));
 
-        $this->register($config['contact_email']);
+        switch ($config['key_type']) {
```

Comment:
This logic is duplicated 3 times, could it be a good idea to put it in a trait? `KeyOptionCommandTrait`?

Reply:
done